substrafl/evaluation_strategy.py

```diff
@@ -18,10 +18,10 @@ def __init__(
 
         Args:
             test_data_nodes (List[TestDataNode]): nodes on which the model is to be tested.
-            eval_frequency (Optional[int]): The model will be tested every ``eval_frequency`` rounds. Default to 1.
-                Set to None to activate eval_rounds only.
+            eval_frequency (Optional[int]): The model will be tested every ``eval_frequency`` rounds.
+                Set to None to activate eval_rounds only. Default to None.
```

Comment:
Nitpick: use the third person to address the variable, like on [line 121](https://github.com/Substra/substrafl/blob/d70bae2d105c6531a6fa0f71989edb7bf5bf52c0/substrafl/evaluation_strategy.py#L121).

```suggestion
                Set to None to activate eval_rounds only. Defaults to None.
```

This also goes for line 24.